src/orders/order_modules/models/order.py

```diff
@@ -29,8 +29,8 @@ class HIBerryOrder(BaseModel):
     cart_items: List[HIBerryProduct]
     total_amount: confloat(ge=0.0)
     payment_method: StrictStr
-    geolocation: Optional[Geolocation]
-    source: Optional[StrictStr]
+    geolocation: Geolocation | None = None
```

Comment:
Under pydantic2 , Optional means _Required, can be None_:https://docs.pydantic.dev/dev/migration/#required-optional-and-nullable-fields